dev/com.ibm.ws.security.oidc.client_fat/publish/files/serversettings/oidcClientBasicConfig_sigAlgNone_withoutSecret.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+	<openidConnectClient
+		id="client01"
+		scope="openid profile"
+		signatureAlgorithm="none"
+		grantType="implicit"
+		clientId="client01"
+		mapIdentityToRegistryUser="true"
+		redirectToRPHostAndPort="https://localhost:${bvt.prop.security_2_HTTP_default.secure}"
+		authorizationEndpointUrl="https://localhost:${bvt.prop.security_1_HTTP_default.secure}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="https://localhost:${bvt.prop.security_1_HTTP_default.secure}/oidc/endpoint/OidcConfigSample/token"
```

Comment:
Technically not needed if the client is using the `implicit` grant type, but shouldn't hurt anything.